mlir/include/mlir/Target/LLVMIR/LLVMImportInterface.h

```diff
@@ -66,6 +75,11 @@ class LLVMImportDialectInterface
   /// returns the list of supported intrinsic identifiers.
   virtual ArrayRef<unsigned> getSupportedIntrinsics() const { return {}; }
 
+  /// Hook for derived dialect interfaces to publish the supported instructions.
+  /// As every LLVM IR instructions has a unique integer identifier, the
+  /// function returns the list of supported instructions identifiers.
+  virtual ArrayRef<unsigned> getSupportedInstructions() const { return {}; }
```

Comment:
What if multiple dialect return overlapping results here?

Also the documentation is saying what this does, but not why it is useful or how this should be used, can you expand?